src/problem2.rb

```diff
@@ -0,0 +1,22 @@
+
+# Given an integer array of size N,
+# identify the largest 2 numbers in this array. Do not sort the array.
+class Problem2
+  # @param [Array] numbers
+  # @return [Array]
+  def self.largest_two_of(numbers)
+    return nil if numbers.nil? || numbers.empty?
+    return numbers if numbers.size == 1
+    first = numbers[0]
+    second = numbers[1]
+    (2..numbers.size - 1).each do |i|
```

Comment:
Very c way to iterate an array in ruby, can you think of a better cleaner way?

TIP: you should not be required to access elements via their index, just have the element available.